server/controllers/issueController.js

```diff
@@ -0,0 +1,26 @@
+import mongoose from 'mongoose';
+import Issue from '../models/issueModel';
+import { buildResponse, joiValidate } from '../utils/helpers';
+import { ISSUE_CHANGE_LIFECYCLE_SCHEMA } from '../utils/constants';
+
+const { ObjectId } = mongoose.Types;
+
+const changeLifeCycle = async (req, res, next) => {
+  const [isValid, response] = joiValidate(req.body, ISSUE_CHANGE_LIFECYCLE_SCHEMA);
+  if (!isValid) return res.status(400).send(response);
+
+  const { _id, lifeCycle } = req.body;
+
+  try {
+    const issue = await Issue.findOneAndUpdate({ _id: ObjectId(_id) }, { $set: { lifeCycle } });
```

Comment:
It does not seem good to expose _id to UI. We can go with generated Id's which would be more readable and short.

Reply:
ok, I'll refactor to that.